pubspec.lock

```diff
@@ -5,10 +5,10 @@ packages:
     dependency: transitive
     description:
       name: _fe_analyzer_shared
-      sha256: f256b0c0ba6c7577c15e2e4e114755640a875e885099367bf6e012b19314c834
+      sha256: "5aaf60d96c4cd00fe7f21594b5ad6a1b699c80a27420f8a837f4d68473ef09e3"
       url: "https://pub.dev"
     source: hosted
-    version: "72.0.0"
+    version: "68.0.0"
```

Comment:
How come we have several versions downgraded? It looks like `app_settings` has no dependencies so it shouldn't affect it.

Reply:
Huh, weird, I'll double-check this

Reply:
ah, running `flutter upgrade` solved this